apps/src/StudioApp.js

```diff
@@ -39,6 +39,7 @@ import project from './code-studio/initApp/project';
 import * as assets from './code-studio/assets';
 import i18n from './code-studio/i18n';
 import AbuseError from './code-studio/components/abuse_error';
+import ObservableEvent from './ObservableEvent';
```

Comment:
Not sure which I prefer, but do you have thoughts on using our own `ObservableEvent` vs Node.js' [`EventEmitter`](https://nodejs.org/api/events.html) (which we're [using in Maker](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/kits/maker/CircuitPlaygroundBoard.js#L36))?